src/matlab2tikz.m

```diff
@@ -1853,13 +1870,14 @@ function assertRegularAxes(handle)
     end
 end
 % ==============================================================================
-function [m2t,str] = plotLine2d(m2t, opts, data)
+function [m2t,str] = plotLine2d(m2t, opts, errorBarOpts, data)
     errorbarMode = (size(data,2) == 4); % is (optional) yDeviation given?
 
     errorBar = '';
     if errorbarMode
         m2t      = needsPgfplotsVersion(m2t, [1,9]);
-        errorBar = sprintf('plot [error bars/.cd, y dir = both, y explicit]\n');
+        errorBar = sprintf('plot [error bars/.cd, y dir = both, y explicit, %s]\n',...
```

Comment:
Continuing with the standardisation, we can make this line just:
`errorBar = sprintf('plot [%s]\n', errorBarOpts);`